app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
The comparison `visitor['vaccine']['expiration_date'] < datetime.date.today()` assumes that `expiration_date` is a `datetime.date` object. Ensure that the `expiration_date` is indeed a `datetime.date` object before this comparison. If it's a string or another type, you may need to convert it to a `datetime.date` object first.